### PR TITLE
lib/nfs_common: Detect if the kernel does not have NFSv2 support

### PR DESCRIPTION
**Pull request by Vogtinator**
- Related ticket: https://progress.opensuse.org/issues/125072
- Verification run: https://openqa.opensuse.org/tests/3147227

### CODE REVIEW

File: lib/nfs_common.pm [hunk 1]
```diff
@@ -29,6 +29,18 @@ sub server_configure_network {
 sub try_nfsv2 {
     # Try that NFSv2 is disabled by default
     systemctl 'start nfs-server';
+
+    # Make sure it exists and also print the content as info.
+    assert_script_run "cat /proc/fs/nfsd/versions";
```
Comment (okurz) [inline lib/nfs_common.pm:34]: you can consider using a `fail_message` which would also be a way to replace the comment with a text string with function

```suggestion
    assert_script_run 'cat /proc/fs/nfsd/versions', fail_message => 'NFS versions file must exist';
```

File: lib/nfs_common.pm [hunk 2]
```diff
@@ -29,6 +29,18 @@ sub server_configure_network {
...
+
+    # It's ok if it's not available in the kernel
+    if (script_run("grep -q '[+-]2' /proc/fs/nfsd/versions") == 1) {
+        record_info("Info", "NFS 2 not available in the kernel");
```
Comment (okurz) [inline lib/nfs_common.pm:38]:
```suggestion
        record_info('Info', 'NFS 2 not available in the kernel');
```

File: lib/nfs_common.pm (hunk 1 above)
Comment (Vogtinator) [reply lib/nfs_common.pm:34]: That's missing the "also print content as info" part

Comment (okurz) [reply lib/nfs_common.pm:34]: well, you get the idea

File: lib/nfs_common.pm (hunk 2 above)
Comment (okurz) [reply lib/nfs_common.pm:38]: you gotta start somewhere :)

Comment (Vogtinator) [reply lib/nfs_common.pm]: Changed.

File: lib/nfs_common.pm (hunk 1 above)
Comment (Vogtinator) [reply lib/nfs_common.pm]: Changed.

**Pull request merged**